Library/Formula/launchdns.rb

```diff
@@ -0,0 +1,55 @@
+class Launchdns < Formula
+  homepage "https://github.com/josh/launchdns"
+  url "https://github.com/josh/launchdns/archive/v1.0.1.tar.gz"
+  head "https://github.com/josh/launchdns.git"
+  sha1 "7310bb558a3b910e98b5364652e3a4fb48375494"
+
+  def install
+    ENV["PREFIX"] = prefix
+    system "make", "install"
+
+    (prefix+"etc/resolver/dev").write("nameserver 127.0.0.1\nport 55353\n")
+  end
+
+  test do
+    system "#{bin}/launchdns", "-p0", "-t1"
+  end
+
+  def caveats; <<-EOS.undent
+      To have *.dev resolved to 127.0.0.1:
+          sudo ln -s #{HOMEBREW_PREFIX}/etc/resolver /etc/resolver
```

Comment:
`/etc/resolver/*` is only used by other user defined TLDs. I think it will be fine since OSX never ships with this directory existing nor uses it itself.